draft-poidt-teas-actn-poi-assurance.txt

```diff
@@ -277,90 +342,446 @@ Table of Contents
+   the IP equipment is configured to operate the reversion
+   automatically, but there are cases where the network operator may
+   prefer differently.
+
+   At the end of the process, multi-layer hitless reversion may take
+   place, again based on the configuration adopted by the network
+   operator.  If multi-layer hitless reversion is adopted, then the
+   process described in Section 7.5 takes place.
+
+7.1.2.  Optical protection
+
+   Differently from the previous case, here optical protection is
+   considered.  This duration of this process is comparable with IP/MPLS
+   FRR, as it is pre-computed.  As a consequence, when multi-layer
+   coordination is enabled it is preferrable to hold-off FRR on R1 and
+   wait that optical protection is completed.  The process is shown in
+   the next figure.
+
+     R1    ROADM1   P-PNC   O-PNC   MDSC   ROADM2    R2    ROADM3    R3
+     |       |1a.Fault notification  |       |       |       |       |
+     |       |-------------->|       |       |       |       |       |
+     |       |       |       |2a.Fault notification  |       |       |
+     |       |       |       |------>|       |       |       |       |
+     |1b.Fault notification  |       |       |       |       |       |
+     |-------------->|       |       |       |       |       |       |
+     |       |       |2b.Fault notification  |       |       |       |
+     |       |       |-------------->|       |       |       |       |
+   ┌――――――┐
+   │3.Hold│
+   └――――――┘
+      ┌―――――――――――――┐
+      │4.Protection │------->|-------------->|
+      └―――――――――――――┘
+     |       |5.Path ready   |   5.Path ready|       |       |       |
+     |       |-------------->|<--------------|       |       |       |
+     |       |       |       |6.Notification |       |       |       |
+     |       |       |       |------>|       |       |       |       |
+   ┌――――――――┐
+   │7.IP Up │
+   └――――――――┘
+
+             Figure 4: Fault detection with optical protection
+
+   The detailed process includes the following steps: 1a.  The fault on
+   the optical path (e.g. fiber cut, loss of signal, etc.) is detected
+   by ROADM1 and notified to O-PNC 2a.  O-PNC notifies the fault to MDSC
+   1b.  R1 detects loss of end-to-end connectivity (e.g. 3 missed BFD
+   messages) and notifies P-PNC.  This step takes place almost
+   simultaneously to 1a. 2b.  P-PNC notifies the issue to MDSC [Editor's
+   note: is this step necessary?] 3.  R1 is configured to hold the FRR
+   process, thus it waits for the corresponding value set by the hold-
+   off time parameter 4.  Optical protection is started by ROADM1,
+   potentially involving an exchange of messages with O-PNC and ROADM2
+   5.  Both ROADM1 and ROADM2 notify O-PNC of the availability of an
+   optical backup path 6.  O-PNC notifies MDSC of the availability of an
+   optical backup path 7.  R1 detects again end-to-end connectivity with
+   R2 and resumes normal IP traffic steering.
```

Comment:
In case of Optical Protection (ie PSM case), there is no way stop and resume the ip traffic. In fact, no impact to IP traffic. Just O-PNC should be aware; it will be a information for it.

Reply:
See https://github.com/Paolo-Volpato/draft-poidt-teas-actn-poi-assurance/commit/bf3de91b1b77ec64af73aaecbddf78bf5870357d